### PR TITLE
ref(ui): Update tag backgrounds & borders

### PR DESCRIPTION
**Pull request by vuluongj20**
Use new, subtler, and more accessible backgrounds and borders for tags. The current design is not accessible (the text-background contrast is 2.9 when it should be at least 4).

Before:
Screen Shot 2021-12-17 at 10 55 50 AM

After:
Screen Shot 2021-12-17 at 10 56 14 AM

Requires https://github.com/getsentry/reload/pull/245

### CODE REVIEW

File: static/app/components/tag.tsx [hunk 1]
```diff
@@ -152,7 +153,10 @@ const IconWrapper = styled('span')`
 `;
 
 const Text = styled('span')<{maxWidth: number; type: keyof Theme['tag']}>`
-  color: ${p => (['black', 'focus'].includes(p.type) ? p.theme.white : p.theme.gray500)};
+  color: ${p =>
+    ['black', 'white'].includes(p.type)
+      ? p.theme.tag[p.type].iconColor
+      : p.theme.textColor};
```
Comment (vuluongj20) [inline static/app/components/tag.tsx:159]: We don't have a `focus` type for tags, and it's not used anywhere in the code.

**Review by davidenwang — APPROVED** [review]

**Comment by vuluongj20** [comment]
@davidenwang Added an analytics event to track how many people would mistakenly click on tags given the new design. Corresponding PR: https://github.com/getsentry/reload/pull/245. Is this enough to create a new analytics event, or am I missing something?

File: static/app/components/tag.tsx [hunk 2]
```diff
@@ -95,6 +96,15 @@ function Tag({
     onDismiss?.();
   }
 
+  const trackClickEvent = () => {
+    trackAnalyticsEvent({
```
Comment (davidenwang) [inline static/app/components/tag.tsx:100]: trackAnalyticsEvent is deprecated we should try and use the trackAdvancedAnalyticsEvent if possible.

**Comment by davidenwang** [comment]
> @davidenwang Added an analytics event to track how many people would mistakenly click on tags given the new design. Corresponding PR: [getsentry/reload#245](https://github.com/getsentry/reload/pull/245). Is this enough to create a new analytics event, or am I missing something?

Yup seems pretty good to me, just left a comment regarding the way the analytics is sent.

**Pull request merged**